<ui version="4.0" >
 <class>PluginSettings</class>
 <widget class="QWidget" name="PluginSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Plugins</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QComboBox" name="comboBox" >
        <item>
         <property name="text" >
          <string>Display Types</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Tools</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Extensions</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Colors</string>
         </property>
        </item>
       </widget>
      </item>
      <item rowspan="2" row="1" column="0" >
       <widget class="Avogadro::PluginListView" name="listView" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Details</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QTextEdit" name="textEdit" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Avogadro::PluginListView</class>
   <extends>QListView</extends>
   <header>pluginlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
